packages/appstore/src/components/wallet-header/wallet-currency-card.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
@sergei-deriv As discussed with the design team today, since we have 2 components related to Wallets; `WalletIcon` and `WalletCard` and 3 variations (S, M and L) each for these two components.
It'll be better if we create these components separately in `components` package and re-use it in `appstore`.

![image (8)](https://user-images.githubusercontent.com/62882794/236467710-5b123cf0-6ee3-48c1-b149-240d25e9878d.png)

Reply:
@aum is already creating `WalletCard` in `components` package. Either of you can implement `WalletIcon`, we can discuss and proceed accordingly.

Reply:
@nijil-deriv Yeah, of course, I'll create WalletIcon. Also I need GradientBackground as this (https://github.com/binary-com/deriv-app/pull/8436/files), so I'll create one more component for gradient fro 2 points and after that I'll create WalletIcon. Is it ok?

Reply:
Yes, that would be great. I think it'll be better if `WalletIcon` is a separate PR since it doesn't have to undergo QA. We can merge it soon and @aum-deriv can use it in his WalletCard component.